libraries/lib-datahandler/src/main/thrift/components.thrift

```diff
@@ -194,6 +196,8 @@ struct Release {
     54: optional COTSDetails cotsDetails,
 
     65: optional set<string> mainLicenseIds,
+    66: optional set<string> mainLicenseNames,
```

Comment:
Not intentionally... It probably came back in the automatic part of the rebase. Removed.